src/doc/rustdoc/src/documentation-tests.md

```diff
@@ -168,37 +168,55 @@ By repeating all parts of the example, you can ensure that your example still
 compiles, while only showing the parts that are relevant to that part of your
 explanation.
 
-Another case where the use of `#` is handy is when you want to ignore
-error handling. Lets say you want the following,
+
+## Using `?` in doc tests
+
+A complete error handling is often not useful in your example, as it would add
```

Comment:
I would change this slightly. This isn't wrong, but sounds a bit strange.

> In your examples, handling all errors is not often useful, as it would add